src/main/groovy/com/profitbricks/sdk/Common.groovy

```diff
@@ -76,6 +77,9 @@ final class Common {
 
             while (true) {
                 final path = loc.toURL().path - URLParts.path
+                if (path.getAt(0) == '/') {
```

Comment:
1. wrong, as path is declared ‘final’
2. also, what is the rationale behind chopping leading slashes?